Description: Don't fail the tests if a cfg cannot be generated for target csky-unknown-linux-gnuabiv2
 This target seems to be present but broken in Ubuntu's rustc.
Author: Peter Michael Green <plugwash@debian.org>
Last-Update: 2024-01-23

--- a/src/resolve.rs
+++ b/src/resolve.rs
@@ -566,7 +566,10 @@
     fn parse_cfg_list() {
         // builtin targets
         for target in cmd!("rustc", "--print", "target-list").read().unwrap().lines() {
-            let _cfg = Cfg::from_rustc(cmd!("rustc"), &target.into()).unwrap();
+            let _cfg = Cfg::from_rustc(cmd!("rustc"), &target.into());
+            if target != "csky-unknown-linux-gnuabiv2" {
+                let _cfg = _cfg.unwrap();
+            }
         }
         // custom targets
         for spec_path in fs::read_dir(fixtures_path().join("target-specs"))
